setup/storage/minio.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: single
+title:  "Minio"
+sidebar:
+  nav: setup
+---
+
+[Minio](https://www.minio.io/) is an S3-compatible object store that you can
+host yourself. This is the persistent storage solution we recommend when you
+don't want to depend on a cloud provider to host your Spinnaker data. However,
+losing Minio's data will mean losing all your Spinnaker application metadata,
+and configured pipelines.
+
+## Prerequisites
+
+Install Minio following the instructions on the [Minio
+homepage](https://www.minio.io/), making sure to have it run on an endpoint
+reachable by Spinnaker. Record the following values:
+
+* `ENDPOINT`: The fully-qualifed endpoint Minio is reachable on. If Minio is
+  running on the same machine as Spinnaker, this might be
+  `http://localhost:9001`.
+
+* `MINIO_ACCESS_KEY`, `MINIO_SECRET_KEY`: The access/secret keypair you've 
+  configured Minio with. These env vars need to be visible to the Minio process
+  for them to work.
+
+## Editing Your Storage Settings
+
+Run the following commands (notice we are picking S3 as our storage type,
+because Minio implements the S3 API):
```

Comment:
Does Minio support other storage types? If not, do we need this "(notice we are...)" ? And if so, should we mention somewhere near the beginning the different storage types it can be used with?

Reply:
My worry is that this looks like a bug/documentation error with `--type s3`. It's not clear why that works, and pointing it out right above the command feels like it might avoid some confusion.

Reply:
Also, it only implements the S3 API AFAIK

Reply:
According to the minio.io/ they're "multi cloud," though they also mention the s3 api (and *only* the s3 api)  right at the top of the page. 
"Minio is an open source object storage server with Amazon S3 compatible API. 
Build cloud-native applications portable across all major public and private clouds"
Do we anticipate anyone using minion who's not using s3? 

I agree, btw, that if you have the `type s3` option and don't point it out, it looks like a mistake.

Reply:
Minio replaces S3, so it's not about which cloud they are using. For example, I know there are subtle difference between S3 and GCS, so while Minio could (in theory) replace GCS, I don't think it's smart to try.